test/ask_web/controllers/respondent_group_controller_test.exs

```diff
@@ -813,17 +813,16 @@ defmodule AskWeb.RespondentGroupControllerTest do
              }
 
       respondents = Repo.all(from r in Respondent, where: r.survey_id == ^survey.id)
+      phone_numbers = respondents |> Enum.map(& &1.phone_number)
 
       assert length(respondents) == 15
 
       assert group
       assert group.respondents_count == 15
-      assert group.sample == respondents |> Enum.take(5) |> Enum.map(& &1.phone_number)
+      assert Enum.count(group.sample) == 5
+      assert Enum.count(group.sample -- phone_numbers) == 0
```

Comment:
Elixir maps don't retain any order, so the original test is no longer working (we can't know which samples have been selected out of the 15). I'm thus checking that the samples come from the actual list of respondents.

TODO: we must still verify that the phone numbers have been masked!